transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql

```diff
@@ -0,0 +1,34 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['event_date']
+    })
+}}
+
+WITH community_prod AS (
+    SELECT 
+        {{dbt_utils.star(ref('stg_mm_telemetry_prod__tracks'))}}
+        , received_at::date as received_at_date
```

Comment:
Not sure why `received_at` is cast to a date here. Is it for incremental logic? If yes, this is complicating the query. 

If it's still needed, [incremental strategy](https://docs.getdbt.com/docs/build/incremental-models#configuring-incremental-strategy) also needs to be configured. In this case, the same `received_at_date` might be part of the query, making the merge strategy the most likely approach.

Reply:
Updated the incremental strategy to append